evaluation/swe_bench/scripts/docker/Dockerfile.builder

```diff
@@ -0,0 +1,17 @@
+FROM ghcr.io/opendevin/sandbox:latest
+
+RUN apt-get update && \
+    apt-get install -y libffi-dev bash gcc git jq wget pkg-config libfreetype-dev libfreetype6 libfreetype6-dev rsync && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN ln -sfn /bin/bash /bin/sh
+RUN mkdir -p /opendevin/logs && chmod 777 /opendevin/logs
+
+# Setup Git
+RUN git config --global user.email "swebench@pnlp.org"
```

Comment:
What is `pnlp.org`?

Reply:
I think this is copied over from their repo.. but this should not matter at all -- let me rebuild it then.